config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -128,6 +132,10 @@ periodics:
 - interval: 1h
   cluster: k8s-infra-prow-build-trusted
   max_concurrency: 1
+  # Run only 1 of image promotion postsubmit and periodic at the same time.
+  # This is an important step to ensure that we avoid issues of running promotion
+  # twice in parallel, such as double signing images.
+  job_queue_name: "k8sio-image-promo"
   # This name is the "job name", passed in as "--job=NAME" for mkpj.
   name: ci-k8sio-image-promo
```

Comment:
I'd like to question if we really need this periodic job. I assume that it's just to be sure that everything is synced, but technically having 2 jobs doing the same thing seems to be an anti-pattern to me.